src/constant/ibc.ts

```diff
@@ -87,14 +87,75 @@ export enum ChainIds {
 	Certik = "shentu-2.2",
 	Carbon = "carbon-1",
 	Axelar = "axelar-dojo-1",
+	Stride = "stride-1",
 }
 
 // whitelisted networks for addition of swth as a currency
 export const swthIbcWhitelist: string[] = [ChainIds.Osmosis];
 // whitelisted networks for addition of transfer options
-export const ibcWhitelist: string[] = [ChainIds.Osmosis, ChainIds.Terra, ChainIds.CosmosHub, ChainIds.Juno, ChainIds.Evmos, ChainIds.Axelar];
+export const ibcWhitelist: string[] = [ChainIds.Osmosis, ChainIds.Terra, ChainIds.CosmosHub, ChainIds.Juno, ChainIds.Evmos, ChainIds.Axelar, ChainIds.Stride];
 
 export const EmbedChainInfosInit: SimpleMap<ChainInfoExplorerTmRpc> = {
+  [ChainIds.Stride]: {
+	rpc: "https://rpc-stride.keplr.app",
+    rest: "https://lcd-stride.keplr.app",
+    chainId: "stride-1",
+    chainName: "Stride",
+    stakeCurrency: {
+      coinDenom: "STRD",
+      coinMinimalDenom: "ustrd",
+      coinDecimals: 6,
+      coinGeckoId: "stride",
+    },
+    bip44: {
+      coinType: 118,
+    },
+    bech32Config: IBCAddress.defaultBech32Config("stride"),
+    currencies: [
+		{
+		  coinDenom: "STRD",
+		  coinMinimalDenom: "ustrd",
+		  coinDecimals: 6,
+		  coinGeckoId: "stride",
+		},
+		{
+		  coinDenom: "stATOM",
+		  coinMinimalDenom: "stuatom",
+		  coinDecimals: 6,
+		},
+		{
+		  coinDenom: "stOSMO",
+		  coinMinimalDenom: "stuosmo",
+		  coinDecimals: 6,
+		},
+		{
+		  coinDenom: "stJUNO",
+		  coinMinimalDenom: "stujuno",
+		  coinDecimals: 6,
+		},
+		{
+		  coinDenom: "stSTARS",
+		  coinMinimalDenom: "stustars",
+		  coinDecimals: 6,
+		},
+	  ],
+	  feeCurrencies: [
+		{
+		  coinDenom: "STRD",
+		  coinMinimalDenom: "ustrd",
+		  coinDecimals: 6,
+		  coinGeckoId: "stride",
+		},
+	  ],
+	  gasPriceStep: {
+		low: 0,
+		average: 0,
+		high: 0.04,
+	  },
+	  features: ["ibc-transfer", "ibc-go"],
```

Comment:
Need to add "no-legacy-stdTx" and "stargate" flags here. If not cannot transfer tokens via IBC.
<img width="657" alt="Screenshot 2022-11-23 at 11 10 27 AM" src="https://user-images.githubusercontent.com/65583522/203462351-2e26fdc8-c971-423c-a561-c52b804a7921.png">